File: LC283.java
class Solution {
    public void moveZeroes(int[] nums) {
        int zeroPos = 0;
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] != 0) {
                nums[zeroPos] = nums[i];
                zeroPos++;
            }
        }
        for (int i = zeroPos; i < nums.length; i++) {
            nums[i] = 0;
        }
    }
}

File: src/LC283.go
func moveZeroes(nums []int)  {
start := 0

for i := 0; i<len(nums); i++{
if nums[i] != 0{
nums[start] = nums[i]
start = start + 1
}

}
for; start<len(nums);start++{
nums[start] = 0
}
}
